src/main/java/com/bieganski/ox/Coordinates.java

```diff
@@ -0,0 +1,42 @@
+package com.bieganski.ox;
+
+import java.util.Objects;
+
+public class Coordinates implements Comparable<Coordinates> {
```

Comment:
Public yet without Javadocs. Can't go in.

Also - why do you need coordinates?